img-src/Dockerfile

```diff
@@ -1,16 +1,10 @@
-FROM debian:buster
+FROM alpine
 
-RUN apt update
-RUN apt upgrade -y
-RUN DEBIAN_FRONTEND=noninteractive apt install -y \
-    bash \
+RUN apk add -U \
     iptables
-RUN apt clean
 
 WORKDIR /app
 
 COPY entrypoint.sh entrypoint.sh
-RUN chmod +x entrypoint.sh
-
```

Comment:
To make `update-alternatives` working, the alternatives must be registered.
This are simple examples that I already tested in an alpine container:
- `update-alternatives --install /sbin/iptables iptables /sbin/iptables-legacy 1`
- `update-alternatives --install /sbin/iptables iptables /sbin/iptables-nft 2`

Reply:
I don't know how to comment on a file without selecting a line, so I commented a random line about this. ;)